admin_guide/master_node_configuration.adoc

```diff
@@ -10,44 +10,116 @@
 toc::[]
 
 == Overview
-`openshift start`, `openshift start master`, and `openshift start node` all take a limited set of arguments that are sufficient for development and experimental purposes, but are insufficient to describe and control the full set of configuration and security options that are necessary in a production environment.  To provide those options, it is necessary to use the dedicated master and node config files.
+The `openshift start` command is used to launch OpenShift servers. The command
+and its subcommands, `master` to launch a
+link:../architecture/infrastructure_components/kubernetes_infrastructure.html#master[master
```

Comment:
I tried to get this long sentence broken into two, but I think it's okay to leave it as one. However, I'd wrap that middle part in parentheses, not commas: "The command and its subcommands (master to launch a master server and node to launch a node server) all take a limited set of arguments that are sufficient for launching servers in a development or experimental environment."